### ISSUE TITLE
Wrong .sentryrccli was used after bundleRelease

### ISSUE THREAD

**Issue by Natteke** [getsentry/sentry-react-native#788]
OS:
- [ ]  Windows
- [ x ]  MacOS
- [ x ]  Linux

Platform:
- [ ]  iOS
- [ x ]  Android

SDK:
- [ x ]  `@sentry/react-native`
- [ ]  `react-native-sentry`

SDK version: 1.3.3
`react-native` version: 0.61.5

Are you using Expo?
- [ ]  Yes
- [ x ]  No
---
I have following issue:
Hi!
After` ./gradlew assembleRelease` sentry tooks `.sentryrccli `from the root user folder instead of project folder.

![image](https://user-images.githubusercontent.com/24633723/76078392-a0d01e80-5fb3-11ea-8dd0-175494b74c6f.png)

**Steps to reproduce:**
1) react-app-init
2) install latest sentry according to the dock, and run wizard

**Actual result:**
sentry takes root `.sentryrccli` from `home/user` folder

**Expected result:**
Sentry take a closest `.sentryclirc` file

Without .sentryrccli sentry will fail sending sourcemaps and fail assembleRelease command.
In most cases this happens if other developer cloned the repo, and has no this file globaly,
and he can't run build.

I am new to sentry, so i'm sorry if i missed something and it is not a bug, but just a behaviour.
But i will be grateful for a solution.

Basically i just need tell which `.sentryrccli` sentry should be used :)

**Comment by Psiiirus** [comment]
Same here to be honest i don't have a .sentryclirc in my project only the "android/sentry.properties" file

**Comment by sboudouk** [comment]
I have the sentry.properties file but with sentry debug set to "info", it seems like it's taking the .sentryclirc... Any reasons ?... I don't even have this file.

`INFO    2020-10-23 16:31:16.449005 +02:00 Loaded config from /Users/sbk/.sentryclirc
  INFO    2020-10-23 16:31:16.449555 +02:00 sentry-cli was invoked with the following command line: "/Users/sbk/Desktop/DMT Content/dmt_ordo/node_modules/@sentry/cli/sentry-cli" "upload-dsym"
error: API request failed
  caused by: sentry reported an error: You do not have permission to perform this action. (http status: 403)`

**Comment by marandaneto** [comment]
is it still an issue using the latest Sentry RN SDK version?
